.idea/runConfigurations.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is probably attached by Intellij, I assume? 
Either way, it is out of scope for this PR/issue and I think it should be removed.

Reply:
Done